tests/tasks/cleanup_candlepin_container.yml

```diff
@@ -0,0 +1,37 @@
+# SPDX-License-Identifier: MIT
+---
+- name: Check if the candlepin container is started
+  command: podman container exists candlepin
+  register: __rhc_candlepin_cont_started
+  failed_when: false
+  changed_when: false
+
+- name: Stop and potentionally remove Candlepin container
+  command: podman stop candlepin
+  when: __rhc_candlepin_cont_started.rc == 0
+  changed_when: true
```

Comment:
Instead of these two tasks, simply use the existing `Stop and remove Candlepin container` task in `setup_candlepin.yml`, as it does exactly that.